src/components/LateralBar/LateralBar.scss

```diff
@@ -20,6 +20,11 @@
         background-color: $darkGreyColor;
         font-size: 1.1rem;
     }
+    .menu-button-actived {
+        color: #2ECA70;
```

Comment:
I would set this color in the vars and import it here. As a general rule, avoid setting colors directly, always use a variable.